### PR TITLE
[#4881] improvement(docs): Add documents about PostgreSQL storage backend

### PR DESCRIPTION
**Pull request by yuqi1129**
### What changes were proposed in this pull request?

Change documents accordingly as we have supported PostgreSQL storage backend. 

### Why are the changes needed?

#4611 lacks related documents.

Fix: #4881 

### Does this PR introduce _any_ user-facing change?

N/A.

### How was this patch tested?

N/A.

### CODE REVIEW

File: docs/how-to-use-relational-backend-storage.md [hunk 1]
```diff
@@ -87,6 +91,81 @@ Finally, you can run the script in the distribution package directory to start t
...
+${GRAVITINO_HOME}/scripts/postgresql/
+```
+
+The script name is like `schema-{version}-postgresql.sql`, and the `version` depends on your Gravitino version.
+For example, if your Gravitino version is the latest release version, then you can choose the **latest version** script.
+If you used a legacy script, you can use `upgrade-{old version}-to-{new version}-postgresql.sql` to upgrade the schema.
+
+### Step 2: Initialize the database
+
+Please create a database and scheam in PostgreSQL in advance, and execute the initialization script obtained above in the database.
```
Comment (xloya) [inline docs/how-to-use-relational-backend-storage.md:119]: scheam -> schema

File: docs/how-to-use-relational-backend-storage.md [hunk 2]
```diff
@@ -87,6 +91,81 @@ Finally, you can run the script in the distribution package directory to start t
...
+
+```
```
Comment (xloya) [inline docs/how-to-use-relational-backend-storage.md:121]: Should add a `shell` or `text` tag here?

File: docs/how-to-use-relational-backend-storage.md [hunk 3]
```diff
@@ -87,6 +91,81 @@ Finally, you can run the script in the distribution package directory to start t
 ./${GRAVITINO_HOME}/bin/gravitino.sh start
 ```
 
-## How to use H2
+## How to use PostgreSQL
+
+### Prerequisites
+
+- PostgreSQL 12 ~ 16. For other versions, the community has not tested fully and may not be compatible.
+- Gravitino distribution package with version `0.6.0` or above.
```
Comment (xloya) [inline docs/how-to-use-relational-backend-storage.md:99]: Do we already support this feature in version 0.6.0?

Comment (yuqi1129) [reply docs/how-to-use-relational-backend-storage.md:99]: Sorry, it's a mistake and Gravitino only support it from 0.7.0.

File: docs/how-to-use-relational-backend-storage.md (hunk 2 above)
Comment (yuqi1129) [reply docs/how-to-use-relational-backend-storage.md:121]: Ok, it's better to do so.

File: docs/how-to-use-relational-backend-storage.md (hunk 1 above)
Comment (yuqi1129) [reply docs/how-to-use-relational-backend-storage.md]: fixed

**Comment by xloya** [comment]
LGTM.

**Review by xloya — APPROVED** [review]

File: docs/gravitino-server-config.md [hunk 4]
```diff
@@ -50,20 +50,20 @@ If you are going to use H2 in the production environment, Gravitino will not gua
...
+| `gravitino.entity.store.versionRetentionCount`    | The Count of versions allowed to be retained, including the current version, used to delete old versions data. Set to at least 1 and no greater than 10.                                                                                                | `1`                              | No                                               | 0.5.0         |
+| `gravitino.entity.store.relational`               | Detailed implementation of Relational storage. `H2`,`MySQL` and `PostgreSQL` is currently supported, and the implementation is `JDBCBackend`.                                                                                                           | `JDBCBackend`                    | No                                               | 0.5.0         |
```
Comment (jerryshao) [inline docs/gravitino-server-config.md:61]: Add a whitespace before `mysql`

**Review by jerryshao — APPROVED** [review]

**Pull request merged**